cli: support `${pid}` placeholder in --cpu-prof-name (#59072)
nodejs/node: doc/api/cli.md

CPU.20190409.202950.15293.0.0.cpuprofile
```

If `--cpu-prof-name` is specified, the provided value is used as a template

> for the file name. Some placeholders are supported and will be substituted
> at runtime:

Reviewer: ```suggestion
for the file name. The following placeholder is supported and will be 
substituted at runtime:
```
Since we only have one placeholder for now, this would be clearer in the singular.

Author: Thanks!
